### PR TITLE
[WFCORE-4314] - verbose and recursive parameter in read-alias(es)

### PR DESCRIPTION
**Pull request by baranowb**
https://issues.jboss.org/browse/WFCORE-4314
Improve read commands for elytron keystore.

### CODE REVIEW

**Comment by bstansberry** [comment]
A version bump is required if one hasn't already been done for the subsystem during dev of the same WF Core .Final release.

**Comment by bstansberry** [comment]
And a transformer is required as the op can be executed on the profile resource.

File: elytron/src/main/java/org/wildfly/extension/elytron/ModifiableKeyStoreDecorator.java [hunk 1]
```diff
@@ -85,8 +85,20 @@ public void registerOperations(ManagementResourceRegistration resourceRegistrati
 
     static class ReadAliasesHandler extends ElytronRuntimeOnlyHandler {
 
+        static final SimpleAttributeDefinition RECURSIVE = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.RECURSIVE, ModelType.BOOLEAN, true)
+                .setAllowExpression(false)
+                .setDefaultValue(new ModelNode(false))
```
Comment (bstansberry) [inline elytron/src/main/java/org/wildfly/extension/elytron/ModifiableKeyStoreDecorator.java]: ModelNode.FALSE

File: elytron/src/main/java/org/wildfly/extension/elytron/ModifiableKeyStoreDecorator.java [hunk 2]
```diff
@@ -85,8 +85,20 @@ public void registerOperations(ManagementResourceRegistration resourceRegistrati
...
+                .build();
+
+        static final SimpleAttributeDefinition VERBOSE = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.VERBOSE, ModelType.BOOLEAN, true)
+                .setAllowExpression(false)
+                .setDefaultValue(new ModelNode(true))
```
Comment (bstansberry) [inline elytron/src/main/java/org/wildfly/extension/elytron/ModifiableKeyStoreDecorator.java]: ModelNode.TRUE

File: elytron/src/main/java/org/wildfly/extension/elytron/ModifiableKeyStoreDecorator.java [hunk 3]
```diff
@@ -115,9 +135,14 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
                 .setAllowExpression(false)
                 .build();
 
+        static final SimpleAttributeDefinition VERBOSE = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.VERBOSE, ModelType.BOOLEAN, true)
+                .setAllowExpression(false)
+                .setDefaultValue(new ModelNode(true))
```
Comment (bstansberry) [inline elytron/src/main/java/org/wildfly/extension/elytron/ModifiableKeyStoreDecorator.java]: ModelNode.TRUE

**Comment by darranl** [comment]
Before asking an engineer for a model bump can we please double check ourselves if one has already been performed (in this case it already has) - those handling reviews tend to be closer to this codebase so can quickly check.

I would also add although very light this really should be handled as an RFE, this is a change in behaviour to public API - most of the steps should be a formality but it needs to the usual plan, test, and docs.

**Comment by baranowb** [comment]
Darn it, missed Darrans comment. So bump to 8 is enough?

**Comment by baranowb** [comment]
This depends on https://github.com/wildfly/wildfly-core/pull/4018

**Pull request closed without merging**